### PR TITLE
Acacia release

### PR DESCRIPTION
**Pull request by tahpot**

### CODE REVIEW

File: docs/extensions/personal-data-bridge.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+        // Do something... like issue a loan based on the customer's trusted credit score!
+    }
+}
+```
+
+## Trusted signers
+
+In the above example we hardcoded an array of addresses (`trustedCreditCompanies`) with a single trusted DID.
+
+`VDAVerificationContract.sol` provies an easier way to manage this from your smart contracts.
```
Comment (aurelticot) [inline docs/extensions/personal-data-bridge.md:260]: Typo "... provides an easier ..."

File: docs/extensions/personal-data-bridge.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    ['uint', 'bytes', 'bytes'],
+    [creditScore, customerCreditRatingProof, creditCompanyContextProof]
+)
+
+// Sign the request parameters with the nonce
+const requestParamsWithNonce = ethers.utils.solidityPack(
+    ['bytes', 'uint'],
+    [requestParams, userDidNonce]
+)
+const signedRequest = await keyring.sign(requestParamsWithNonce)
```
Comment (aurelticot) [inline docs/extensions/personal-data-bridge.md:203]: Should be `customerKeyring`

File: docs/extensions/personal-data-bridge.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+////////////////////////////////////////////////////////////////////////////////////////////////////
+// Finally, we submit the request to the smart contract
+await contract.submitCreditScore(getAddressFromDid(did), requestParams, signedRequest, customerContextProof)
```
Comment (aurelticot) [inline docs/extensions/personal-data-bridge.md:207]: Which `did` it is?
I guess `customerDid`?

Comment (tahpot) [reply docs/extensions/personal-data-bridge.md:209]: Yep, fixed.

File: docs/extensions/personal-data-bridge.md (hunk 2 above)
Comment (tahpot) [reply docs/extensions/personal-data-bridge.md:205]: Fixed.

File: docs/extensions/personal-data-bridge.md (hunk 1 above)
Comment (tahpot) [reply docs/extensions/personal-data-bridge.md:262]: Fixed

**Review by aurelticot — CHANGES_REQUESTED** [review]
In addition to the inline comments, there were a bunch of comments I made in a [previous PR](https://github.com/verida/documentation/pull/131/files) that were not taken into account. Any chance we can fix the typos and other suggestions?

File: docs/concepts/data-storage.md [hunk 4]
```diff
@@ -29,8 +76,29 @@ Under the hood a combination of [CouchDB](https://en.wikipedia.org/wiki/Apache_C
 
 At the end of the day, data must be physically stored somewhere. Verida Client SDK is designed to empower user’s to control where their data is stored. This meets the principle: `Put the user first`.
 
-## Data synchronisation
+## How is data syncronized between applications?
```
[the remark this answers is not in the document]
Comment (nick-verida) [reply docs/concepts/data-storage.md:79]: No this is a typo

File: docs/concepts/data-storage.md [hunk 5]
```diff
@@ -1,22 +1,69 @@
...
 
-In this way, a simple onboarding exists for new users, while more advanced users can host their own data.
+[Learn more: Application Contexts](./application-contexts)
+
+## How is data guaranteed to be available?
 
-## How data is stored
+User data is replicated across multiple storage nodes on the Verida network. By default the Verida Wallet selects three storage nodes to store user data; providing three replicas of all the data. The network architecture allows users to select an unlimited number of storage nodes to store their data (and which nodes).
+
+This ensures tht if one of the storage nodes becomes unavailable, there are two other nodes that still have copies of the data and the user can continue uninterupted.
```
[the remark this answers is not in the document]
Comment (nick-verida) [reply docs/concepts/data-storage.md:62]: uninterupted -> uninterrupted

File: docs/concepts/accounts-and-identity.md [hunk 6]
```diff
@@ -6,38 +6,51 @@ keywords: [Verida, Web3, Developers]
...
-For simplicity, Verida provides a free testnet database that can be used by any application, however both application developers and users can take control of this data:
+:::caution
 
-1. Application developers can host their own Verida Storage node(s) and configure that to be used for their application. See [storage nodes](../network/storage-node.md).
-2. Users can host their own Verida Storage node(s) and force an application to use that server by updating their DID document with the new server endpoint
+Data stored on the Verida Acacia Testnet is deleted regularly, so don’t rely on that data persisting.
 
-<aside>
-💡 Data stored on the Verida testnet is deleted regularly, so don’t rely on that data persisting. If you need help spinning up your own dataserver for your application, please contact us.
+If you need help spinning up your own storage node, please contact us.
```
[the remark this answers is not in the document]
Comment (nick-verida) [reply docs/concepts/accounts-and-identity.md:54]: I think there is a contact form, linking to https://discord.verida.io would also work

**Comment by nick-verida** [comment]
This change has a bunch of broken links. The GitHub action shows errors, and there are a number of warnings

Run `yarn build` (**not** just `yarn start`)  to check them:

[ERROR] Error: Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

- On source page path = /docs/client-sdk/how-it-works:
   -> linking to ../network/storage-node.md (resolved as: /docs/network/storage-node.md)

- On source page path = /docs/concepts/accounts-and-identity:
   -> linking to Authentication (resolved as: /docs/concepts/Authentication)

- On source page path = /docs/concepts/network-architecture:
   -> linking to ../network/storage-node.md (resolved as: /docs/network/storage-node.md)

- On source page path = /docs/intro:
   -> linking to /docs/demos

- On source page path = /docs/single-sign-on-sdk:
   -> linking to the-sso-authentication-server.md (resolved as: /docs/the-sso-authentication-server.md)

- On source page path = /docs/single-sign-on-sdk/sso-authentication-server:
   -> linking to ./single-sign-on-sdk/getting-started#configuration (resolved as: /docs/single-sign-on-sdk/single-sign-on-sdk/getting-started)


[WARNING] Docs markdown link couldn't be resolved: (../network/storage-node.md) in /Users/nlothian/dev/verida/documentation/docs/concepts/network-architecture.md for version current
[WARNING] Docs markdown link couldn't be resolved: (the-sso-authentication-server.md) in /Users/nlothian/dev/verida/documentation/docs/single-sign-on-sdk/single-sign-on-sdk.md for version current
[WARNING] Docs markdown link couldn't be resolved: (the-sso-authentication-server.md) in /Users/nlothian/dev/verida/documentation/docs/single-sign-on-sdk/single-sign-on-sdk.md for version current
[WARNING] Docs markdown link couldn't be resolved: (../network/storage-node.md) in /Users/nlothian/dev/verida/documentation/docs/concepts/network-architecture.md for version current
[WARNING] Docs markdown link couldn't be resolved: (../network/storage-node.md) in /Users/nlothian/dev/verida/documentation/docs/client-sdk/how-it-works.md for version current
[WARNING] Docs markdown link couldn't be resolved: (../network/storage-node.md) in /Users/nlothian/dev/verida/documentation/docs/client-sdk/how-it-works.md for version current

**Review by nick-verida — CHANGES_REQUESTED** [review]
There are lots of typos that need fixing (maybe install https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker - I assume it works for Markdown)

Also a number of broken links which stop it from building. 

Run `yarn build` to check and fix.

File: docs/concepts/data-storage.md [hunk 7]
```diff
@@ -29,8 +76,29 @@ Under the hood a combination of [CouchDB](https://en.wikipedia.org/wiki/Apache_C
...
 
-Data from multiple applications using the same schemas can be synchronized automating conflict management. This works in a similar way to a `git merge`, but for database data.
\ No newline at end of file
+Data from multiple applications using the same schemas can be synchronized automating conflict management. This works in a similar way to a `git merge`, but for database data.
+
+The Verida Wallet provides an interface for requesting access to sharing data and automatically handles creating these data syncronization rules between applications.
+
+[Learn more: Data Sharing](./data-sharing)
+
+## How do Verida Accounts authenticate with Storage Modes?
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:89]: Fixed

File: docs/concepts/data-storage.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:79]: Fixed

File: docs/concepts/data-storage.md [hunk 8]
```diff
@@ -1,22 +1,69 @@
...
+
+There are a range of decentralized data storage options for Web3, however they suffer from some fundamental drawbacks:
+
+1. No built-in encryption for private data
+2. Users have no control over where their data is stored
+3. Users have no ability to "self-host" their data
+4. Not GDPR compliant
+5. Poor performance (much slower than Web2 storage)
+
+Verida solve all these data storage problems in a way that is capable of scaling to billions of users and enable decentralized apps to be built that can compete directly with Web2 applications.
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:18]: Fixed. Updated wording.

File: docs/concepts/data-storage.md [hunk 9]
```diff
@@ -1,22 +1,69 @@
...
+
+## Where is data stored?
+
+![Verida Data Storage Network](data_storage/data-storage-network.png)
+
+User data is stored on Verida [Storage Nodes](../infrastructure/storage-node) connected to the Verida network. This offers a publicly available, decentralized, globally distributed collection of encrypted storage nodes.
+
+The Verida network architecture allows users to determine which nodes on the network store their data, or even self-host and use their own private nodes.
+
+Users grant access to authentications to access and decrypt portions of their data stored on nodes across the network.
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:32]: Fixed, should be "grant access to applications"

File: docs/concepts/data-storage.md [hunk 10]
```diff
@@ -1,22 +1,69 @@
...
+
+It is currently possible to manage this via our client SDK, but that is not a suitable user experience for the vast majority of users. In the future the Verida Wallet will provide a user interface allowing users to easily manage and control where their data is stored.
+
+## How is data kept secure?
+
+User data is encrypted using private keys stored on the end user's device. For example, if a user logs into an application in their web browser, an encryption key exists in the browser to encrypt and decrypt all their data. Any data sent to a storage node has already been encrypted and can't be decrypted without the user's key.
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:38]: Good point. Have updated the wording, is that better?

File: docs/concepts/data-storage.md [hunk 11]
```diff
@@ -1,22 +1,69 @@
...
+
+In addition, storage nodes only accept requests that have been authenticated by the owner of the data. Unlike other Web3 storage solutions where all data is publicly readable, the Verida network has both access controls (read, write, delete) in addition to the built-in encryption.
+
+This provides Web2 level security to user data for Web3.
+
+[Learn more: Client SDK / Permissions](../client-sdk/permissions)
+
+## How is application data kept separate from each other?
+
+Verida Accounts have multiple containers for storing data (called [application contexts](./application-contexts)). Each of these `application contexts` are completely independent. They have independent storage nodes storing their data, encryption keys with the data completely isolated from other `application contexts`.
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:48]: The architecture allows them to be different, but by default the Verida Wallet uses the same ones for every context.

File: docs/concepts/data-storage.md [hunk 12]
```diff
@@ -1,22 +1,69 @@
...
+
+1. No built-in encryption for private data
+2. Users have no control over where their data is stored
+3. Users have no ability to "self-host" their data
+4. Not GDPR compliant
+5. Poor performance (much slower than Web2 storage)
+
+Verida solve all these data storage problems in a way that is capable of scaling to billions of users and enable decentralized apps to be built that can compete directly with Web2 applications.
+
+We have written an overview of the types of Web3 data that provide helpful background information and context for where the Verida network offers the biggest value as a data storage solution.
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/data-storage.md:20]: Agreed. Updated.

File: docs/concepts/accounts-and-identity.md [hunk 13]
```diff
@@ -6,38 +6,51 @@ keywords: [Verida, Web3, Developers]
...
 
-Users can have multiple identities depending on context. For example, a user may have a `personal` identity and a `professional` identity. This is easily supported (and free) as a user can simply create a new Verida account (DID) and use that to login to any application.
+Users can have multiple identities for different purposes. For example, a user may have a `personal` identity and a `professional` identity. This is easily supported (and free) as a user can simply create a new Verida account (DID) and use that to login to any application.
 
 It’s also entirely anonymous. No information is known about the holder of the DID and no personally identifiable information is written to a public blockchain.
 
 ## DID Registry
 
-Verida operates a centralized DID registry as a REST server ([Verida DID server](https://github.com/verida/did-server)).
+DID's are more than just an address. The specification supports creating DID Documents that provide enhanced discovery of services being operated for that DID (ie: storage, messaging) and public keys (useful for verying data signed by the DID).
```
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/accounts-and-identity.md:27]: Fixe

File: docs/concepts/accounts-and-identity.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (tahpot) [reply docs/concepts/accounts-and-identity.md:54]: Added both.

File: docs/concepts/data-storage.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (aurelticot) [reply docs/concepts/data-storage.md]: You have to check the "h" key of your keyboard @tahpot :-)

File: docs/concepts/data-storage.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (aurelticot) [reply docs/concepts/data-storage.md:48]: Applications could use other Storage nodes than the ones defined by the user?

File: docs/concepts/data-storage.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (aurelticot) [reply docs/concepts/data-storage.md]: Yep

**Review by aurelticot — APPROVED** [review]

File: docs/single-sign-on-sdk/single-sign-on-sdk.md [hunk 14]
```diff
@@ -5,24 +5,20 @@ description: Verida Developer Documentation
...
+It is an easy-to-use integration method that allows a user to scan a QR code to sign into your application. If the user doesn't have the Verida Wallet installed, they will be prompted to install it. Existing users will be prompted to authorize your application to access encrypted storage for that 
 application.
 
 ### How it works
 
-A user downloads the Verida Vault onto their mobile device which generates a private key on a supported blockchain. This private key is used to generated a Decentralized Identity (DID). The user then scans a QR code on their phone which prompts the user to sign a consent message using the Verida Vault to unlock the secure enclave for that application.
+A user downloads the Verida Wallet onto their mobile device which generates a private key on a supported blockchain. This private key is used to generated a Decentralized Identity (DID). The user then scans a QR code on their phone which prompts the user to sign a consent message using the Verida Wallet to unlock the secure enclave for that application.
```
Comment (aurelticot) [inline docs/single-sign-on-sdk/single-sign-on-sdk.md:17]: Typo "...is used to generate a Decentralized..."

**Review by aurelticot — CHANGES_REQUESTED** [review]

File: docs/single-sign-on-sdk/single-sign-on-sdk.md (hunk 14 above)
Comment (nick-verida) [reply docs/single-sign-on-sdk/single-sign-on-sdk.md]: I've pushed a change to fix this

**Comment by nick-verida** [comment]
Looks good here. 

I'm merging to `develop`

**Pull request merged**